gstd/gstd_pipeline.c

```diff
@@ -199,6 +207,11 @@ gstd_pipeline_class_init (GstdPipelineClass * klass)
       G_GINT64_CONSTANT(0), /* Default value */
       G_PARAM_READABLE | G_PARAM_STATIC_STRINGS);
 
+  properties[PROP_VERBOSE] =
+      g_param_spec_boolean ("verbose", "Verbose",
+      "Verbose state",
```

Comment:
Make explicit that the verbose state is for the pipeline at the blurb.
`Verbose state for the media stream pipeline`.